content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -156,7 +156,7 @@ already be running.
 You can view running containers (including static Pods) by running (on the node):
 ```shell
 # Run this command on the node where the kubelet is running
-docker ps
+crictl ps
```

Comment:
Does the switch to `crictl` change the output format? If so, we should revise that too.

Reply:
The image name is excessively long. Do we have a convention to handle that? Example:  
\`docker.io/library/nginx@sha256:0d17b565c37bcbd895e9d92315a05c1c3c9a29f762b011a10c54a66cd53c9b31 \`